.jvmopts

```diff
@@ -1 +1,5 @@
--XX:-TieredCompilation
\ No newline at end of file
+-XX:-TieredCompilation
```

Comment:
no need to disable tiered compilation, sbt will use only the config in this file. I guess it doesn't hurt.